packages/ws-worker/src/start.ts

```diff
@@ -72,6 +78,12 @@ const args = yargs(hideBin(process.argv))
     default: 5,
     type: 'number',
   })
+  .option('state-props-to-remove', {
```

Comment:
@taylordowns2000 you can now override `statePropsToRemove` from a worker arg or env. It's quite a verbose name but I'm not sure I'd want to make it anything shorter :thinking: